src/php/binaries.rs

```diff
@@ -76,25 +78,38 @@ pub(crate) fn all() -> HashMap<PhpVersion, PhpBinary> {
 
 fn get_all() -> HashMap<PhpVersion, PhpBinary> {
     let mut binaries: HashMap<PhpVersion, PhpBinary> = HashMap::new();
+    let os_infos = os_info::get();
+    let os_version = os_infos.version();
+    let bigsur = "11.0.0";
 
-    binaries_from_env(&mut binaries);
     binaries_from_rymfony_env(&mut binaries);
 
-    merge_binaries(&mut binaries, binaries_from_dir(PathBuf::from("/usr/bin")));
-    merge_binaries(&mut binaries, binaries_from_dir(PathBuf::from("/usr/sbin")));
+    if cfg!(not(target_os = "macos")) || (cfg!(target_os = "macos") && VersionCompare::compare_to(&os_version.to_string(), &bigsur, &CompOp::Lt).unwrap()) {
```

Comment:
Could you move this logic into a specific function, like `fn is_macos_big_sur()` and add some comments about why we're doing this hacky check?

Reply:
Done ! Sorry for the style, I don't know rust 😅

Reply:
Np, it's not related to Rust, it's more about tidying the code 😁 You did well though!